runtime/flang/curdir.c

```diff
@@ -102,5 +104,8 @@ void __fort_gethostname(host) char *host;
     }
     p = un.nodename;
   }
+#else
+  strcpy(p, "localhost");
```

Comment:
There should be an equivalent for `gethostname` in the Win32 or Winsock API, right?

Can you keep the support for `-curhost` on Windows?

Reply:
There is, but I haven't put it in yet.

Reply:
that should be good for now; I will add support for this later.

Reply:
Will add TODO in a separate patch.